corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -248,6 +247,11 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
             } else if (this.model.get('input') === 'address') {
                 // geocoderItemCallback sets the value on the model
             } else {
+                if (!this.isValid()) {
+                    FormplayerFrontend.trigger('showInputError', this.model.get("error") ||
+                    "Please enter a value for this field.", this.model.get("text"));
```

Comment:
is this text translated somewhere? (same with below as well)

Reply:
It's not, but it will be, thanks.